File: Programming EF Code First/chapter4/Chapter4/BAGA_DataAnnotations/DataAccess/BreakAwayContext.cs
using System.Data.Entity;
using Model;

namespace DataAccess
{
  public class BreakAwayContext : DbContext
  {
    public DbSet<Destination> Destinations { get; set; }
    public DbSet<Lodging> Lodgings { get; set; }
    public DbSet<Trip> Trips { get; set; }
    public DbSet<Person> People { get; set; }
  }
}

File: Programming EF Code First/chapter5/Chapter 5 VB/Data Annotations/DataAccess/BreakAwayContext.vb
Imports System.Data.Entity
Imports Model

Namespace DataAccess
  Public Class BreakAwayContext
    Inherits DbContext
    Public Property Destinations() As DbSet(Of Destination)
    Public Property Lodgings() As DbSet(Of Lodging)
    Public Property Trips() As DbSet(Of Trip)
    Public Property People() As DbSet(Of Person)
  End Class
End Namespace